modules/isescan.nf

```diff
@@ -1,45 +1,42 @@
 #!/usr/bin/env nextflow
 nextflow.enable.dsl=2
 
-process isescan {
-    publishDir "$launchDir/"
+process ISESCAN {
+    publishDir "$launchDir/$params.outdir/prediction"
 
     cpus 8
     memory { 8.GB * task.attempt }
     errorStrategy 'retry' 
-    maxRetries 4
+    maxRetries 3
 
-    container "quay.io/microbiome-informatics/isescan-v1.7.2.3"
+    container 'quay.io/microbiome-informatics/isescan-v1.7.2.3'
 
     input:
-        path assembly_file, name: 'contigs.fasta'
+        path assembly_file
 
     output:
-        path 'isescan_results/contigs.fasta.is.fna', emit: iss_fasta 
-        path 'isescan_results/contigs.fasta.tsv', emit: iss_tsv
+        path 'isescan_results/1kb_contigs.fasta.tsv', emit: iss_tsv
 
     script:
     if (assembly_file.size() > 0)
         """
-        isescan.py --seqfile contigs.fasta \
+        isescan.py --seqfile ${assembly_file} \
         --output isescan_results \
         --nthread ${task.cpus}
 
-        if ls -l isescan_results/contigs.fasta.tsv 2>/dev/null | grep -q .
+        if ls -l isescan_results/1kb_contigs.fasta.tsv 2>/dev/null | grep -q .
```

Comment:
Is this in case the module is executed several times (as in retries)?

Reply:
No, it is in case there's no output file, we generate a dummy file to pass it to the integrator step